packages/v/vulkan-volk/package.yml

```diff
@@ -0,0 +1,28 @@
+name       : vulkan-volk
+version    : 1.3.275
+release    : 1
+source     :
+    # Commit from where they updated the version to match the current Vulkan version (keep in sync)
+    - git|https://github.com/zeux/volk.git : b0a52281ee98fe0fb8ff1f5dfe221096aa6dad42
+license    : MIT
+component  : xorg.display
+homepage   : https://github.com/zeux/volk
+summary    : Meta loader for Vulkan API
+description: |-
```

Comment:
Stray hyphen

Reply:
Actually intentional. It removes the trailing newline after the description in the pspec.

Reply:
I see. I was wondering whether it was good for something. So we should probably standardise on this way?

Reply:
It would be nice to do so. For instance if you take a look at the doxygen pspec you'll see that there's a newline after the description, using `|-` gets rid of that.